draft-ietf-mls-protocol.md

```diff
@@ -1207,15 +1209,15 @@ the signature on the message,  then updates its state as follows:
 The update secret resulting from this change is an all-zero octet
 string of length Hash.length.
 
-On receipt of an Add message, new participants SHOULD send an update
-immediately to their key. This will help to limit the tree structure
+After processing an Add message, new members SHOULD send an Update
```

Comment:
This would read better as singular, "... the new member SHOULD..."

Reply:
Fixed